parser/src/cfg/platform.rs

```diff
@@ -138,7 +138,7 @@ pub(crate) fn filter_env_specific_cfg(
                 env_var_val.is_empty(),
             ) {
                 (None, false) => {
-                    lsp_hint_inactive_code.push(LspHintInactiveCode {
+                    lsp_hint_inactive_code.push(lsp_hints::InactiveCode {
```

Comment:
Can we add the conditional compilation with LSP here as well?

Reply:
0b1bf3db8435d3e0e53632168713ad320186dd1e